### PR TITLE
Remove Kobold API style, Added custom to the Loading Models Dropdown, Remove chatgpt preset

### PR DESCRIPTION
**Pull request by ItsSimko**
## Summary by Sourcery

Enhancements:
- Removed the API Style dropdown
- Removed ChatGPT to further explain how to connect to the documentation. #166
-  Added a custom option to the models list dropdown.
- Disable architecture select on remote useage

### CODE REVIEW

File: src/FreeScribe.client/client.py [hunk 1]
```diff
@@ -767,25 +767,34 @@ def send_text_to_api(edited_text):
...
-
-            verify = not app_settings.editable_settings["AI Server Self-Signed Certificates"]
-            response = requests.post(app_settings.editable_settings["Model Endpoint"] + "/api/v1/generate", json=prompt, verify=verify)
-
-            if response.status_code == 200:
-                results = response.json()['results']
-                response_text = results[0]['text']
-                response_text = response_text.replace("  ", " ").strip()
-                return response_text
+        # Open API Style
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/client.py:770]: I know Michael you mentioned to comment it out. Let me know if you want the comment removed or left in the code.

File: src/FreeScribe.client/UI/SettingsWindowUI.py [hunk 2]
```diff
@@ -297,14 +305,28 @@ def toggle_remote_llm_settings(self):
...
+        self.architecture_dropdown.config(state=inverted_state)
+
+        # if local llm is now disabled do what we need to aka update the models
+        if current_state == 0:
+            threading.Thread(target=self.settings.update_models_dropdown, args=(self.models_drop_down,self.settings.editable_settings_entries["Model Endpoint"].get(),)).start()
+            self.on_model_selection_change(None)
+        else:
+            # when turned on again provide local models
+            threading.Thread(target=self.settings.update_models_dropdown, args=(self.models_drop_down,self.settings.editable_settings_entries["Model Endpoint"].get(),)).start()
+            self.on_model_selection_change(None)
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/UI/SettingsWindowUI.py]: 9ee914dbdea90f79d0420d21274960e29cd5be82

File: src/FreeScribe.client/UI/SettingsWindow.py [hunk 3]
```diff
@@ -400,9 +399,11 @@ def get_available_models(self):
             "X-API-Key": self.OPENAI_API_KEY
         }
 
+        endpoint = endpoint or self.editable_settings_entries["Model Endpoint"].get()
+
         try:
             verify = not self.editable_settings["AI Server Self-Signed Certificates"]
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/UI/SettingsWindow.py:415]: f84e814313e68e90c7c83defe55acf46222521e9

File: src/FreeScribe.client/client.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (yingbull) [reply src/FreeScribe.client/client.py:770]: Commented (unless logic gate works) - but this is planned for revision so sourcery, make an issue ticket to track

**Review by yingbull — APPROVED** [review]

**Pull request merged**